src/cider/type/plan/UnaryExpr.cpp

```diff
@@ -25,50 +25,52 @@ namespace Analyzer {
 using namespace cider::jitlib;
 
 JITValuePointer codegenCastBetweenDateAndTime(CodegenContext& context,
-                                              JITValue& operand_val,
+                                              JITValuePointer operand_val,
                                               const SQLTypeInfo& operand_ti,
                                               const SQLTypeInfo& ti) {
-  JITFunction& func = *context.getJITFunction();
```

Comment:
Hmm, why changed this from reference to pointer? You needn't to change anything following if using ```JITFunction& func```.